app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -42,6 +42,7 @@
 
         <org.oppia.android.app.customview.LessonThumbnailImageView
           android:id="@+id/chapter_thumbnail"
+          android:alpha="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? 1.0f : 0.3f}"
```

Comment:
Nit formatting, you can also use Reformat code from the menu in the toolbar or double shift to open quick search in android studio to find reformat code option. 

```xml
 <org.oppia.android.app.customview.LessonThumbnailImageView
          android:id="@+id/chapter_thumbnail"
          android:layout_width="match_parent"
          android:layout_height="0dp"
          android:alpha="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? 1.0f : 0.3f}"
          android:scaleType="centerInside"
          app:entityId="@{viewModel.storyId}"
          app:entityType="@{viewModel.entityType}"
          app:layout_constraintDimensionRatio="16:9"
          app:layout_constraintEnd_toEndOf="parent"
          app:layout_constraintStart_toStartOf="parent"
          app:layout_constraintTop_toTopOf="parent"
          app:lessonThumbnail="@{viewModel.chapterThumbnail}" />
```

Reply:
I ran Reformat code, but I don't think anything changed.

Reply:
It will rearrange the lines above.

Reply:
Ah, to arrange alphabetically? I removed `android:alpha` line, so it should be ok now.

Reply:
It's like arrange things alphabetically, spacing, empty line removals, all this will be done automatically with the reformat code option. 

Also, I suggest if you could set up macros, it will help you a lot through the project for all the files. 
Check the wiki guide here - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

![Screenshot 2020-12-12 at 09 18 54](https://user-images.githubusercontent.com/25057618/101973045-13fd7400-3c5b-11eb-8b1e-530aeb908b6c.png)

Reply:
Thanks.